.pre-commit-config.yaml

```
@@ -0,0 +1,13 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 21.7b0
+    hooks:
+      - id: black
+  - repo: https://github.com/pycqa/flake8
+    rev: "3.9.2"
+    hooks:
+      - id: flake8
+  - repo: https://github.com/pycqa/isort
+    rev: "5.8.0"
+    hooks:
+      - id: isort
```

Address the isort conflicts with black https://github.com/PyCQA/isort/issues/1518
```suggestion
      - id: isort
      - args: ["--profile", "black"]
```

---

Not needed, settings specified in `pyproject.toml`